docs/content/references/metrics/agent.md

```diff
@@ -0,0 +1,132 @@
+
+## RateLimiter
+
+### Metrics
+
+| Name                 | Type    | Labels                                    | Unit            | Description                                                        |
+| -------------------- | ------- | ----------------------------------------- | --------------- | ------------------------------------------------------------------ |
+| rate_limiter_counter | Counter | policy_name, policy_hash, component_index | count (no unit) | A counter measuring the number of times Rate Limiter was triggered |
+
+### Labels
+
+| Name            | Example                                      | Description                                                     |
+| --------------- | -------------------------------------------- | --------------------------------------------------------------- |
+| policy_name     | service1-demo-app                            | Name of the policy.                                             |
+| policy_hash     | 5kZjjSgDAtGWmLnDT67SmQhZdHVmz0+GvKcOGTfWMVo= | Hash of the policy used for checking integrity of the policy.   |
+| component_index | 13                                           | Index of the component in order of specification in the policy. |
+
+## Classifier
+
+### Metrics
+
+| Name               | Type    | Labels                                     | Unit            | Description                                                      |
+| ------------------ | ------- | ------------------------------------------ | --------------- | ---------------------------------------------------------------- |
+| classifier_counter | Counter | policy_name, policy_hash, classifier_index | count (no unit) | A counter measuring the number of times classifier was triggered |
+
+### Labels
+
+| Name             | Example                                      | Description                                                      |
+| ---------------- | -------------------------------------------- | ---------------------------------------------------------------- |
+| policy_name      | service1-demo-app                            | Name of the policy.                                              |
+| policy_hash      | 5kZjjSgDAtGWmLnDT67SmQhZdHVmz0+GvKcOGTfWMVo= | Hash of the policy used for checking integrity of the policy.    |
+| classifier_index | 0, 1                                         | Index of the classifier in order of specification in the policy. |
+
+## Dataplane Summary
+
+### FlowControl Metrics
+
+| Name                             | Type    | Labels        | Unit            | Description                                     |
+| -------------------------------- | ------- | ------------- | --------------- | ----------------------------------------------- |
+| flowcontrol_requests_total       | Counter |               | count (no unit) | Total number of aperture check requests handled |
```

Comment:
@hasit,
All of the labels are not applicable to each metric. Please refer the labels for each metric carefully: https://github.com/fluxninja/aperture/blob/72ad678cc64ddedfd53498b0a4eef0b6cb65bf24/pkg/policies/flowcontrol/api/base/metrics.go#L55

Please do this for all the other metrics as well

Reply:
Done. Confirmed and updated the tables.